plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -1572,6 +1606,42 @@ var _ = Describe("validator", func() {
 
 					Expect(shoot.Spec.Provider.Workers[0].Kubernetes.Version).To(Equal(pointer.String("1.18.5")))
 				})
+
+				It("should reject to create a cluster with an expired worker group kubernetes version", func() {
+					deprecatedClassification := core.ClassificationDeprecated
+					expiredKubernetesVersion := "1.18.1"
+					validKubernetesVersion := "1.18.3"
+					shoot.Spec.Kubernetes.Version = validKubernetesVersion
+					shoot.Spec.Provider.Workers[0].Kubernetes = &core.WorkerKubernetes{Version: &expiredKubernetesVersion}
+					cloudProfile.Spec.Kubernetes.Versions = append(cloudProfile.Spec.Kubernetes.Versions, core.ExpirableVersion{Version: expiredKubernetesVersion, Classification: &deprecatedClassification, ExpirationDate: &metav1.Time{Time: metav1.Now().Add(time.Second * -1000)}}, core.ExpirableVersion{Version: validKubernetesVersion})
+
+					Expect(coreInformerFactory.Core().InternalVersion().Projects().Informer().GetStore().Add(&project)).To(Succeed())
+					Expect(coreInformerFactory.Core().InternalVersion().CloudProfiles().Informer().GetStore().Add(&cloudProfile)).To(Succeed())
+					Expect(coreInformerFactory.Core().InternalVersion().Seeds().Informer().GetStore().Add(&seed)).To(Succeed())
+					attrs := admission.NewAttributesRecord(&shoot, nil, core.Kind("Shoot").WithVersion("version"), shoot.Namespace, shoot.Name, core.Resource("shoots").WithVersion("version"), "", admission.Create, &metav1.CreateOptions{}, false, nil)
+
+					err := admissionHandler.Admit(context.TODO(), attrs, nil)
+
+					Expect(err).To(MatchError(ContainSubstring(fmt.Sprintf("spec.provider.workers[0].kubernetes.version: Unsupported value: \"%s\"", expiredKubernetesVersion))))
```

Comment:
```suggestion
					Expect(err).To(MatchError(ContainSubstring("spec.provider.workers[0].kubernetes.version: Unsupported value: %q", expiredKubernetesVersion)))
```

Reply:
Fixed, thanks!

Reply:
This suggestion is not completely resolved - you can omit the `fmt.Sprintf` and use the same functionality supported by `ContainSubstring`.